Gemfile

```diff
@@ -6,3 +6,6 @@ if RUBY_VERSION < "1.9" && !%w{jruby rbx}.include?(ruby_engine)
   # Built and installed via ext/mkrf_conf.rb
   gem 'system_timer', '~> 1.2'
 end
+
+# fixes 2.3.0 ffi bundle error
+gem 'ffi', '~> 1.0.11'
```

Comment:
This is uh… an extremely old version of ffi (2011). Can we lock to `~> 1.0`, or does it have to be 1.0.11?